src/openforms/forms/models/form.py

```diff
@@ -64,7 +64,10 @@ class Form(models.Model):
         _("slug"), max_length=100, populate_from="name", editable=True, unique=True
     )
     product = models.ForeignKey(
-        "products.Product", null=True, blank=True, on_delete=models.CASCADE
+        "products.Product",
+        null=True,
+        blank=True,
+        on_delete=models.CASCADE,
     )
```

Comment:
Black behaving weirdly

Reply:
I'm going to guess that you have a different version than the one pinned in the requirements?

Reply:
Weird, still have the correct one (both styles are valid with black anyway).